src/node_sea.cc

```diff
@@ -0,0 +1,126 @@
+#include "node_sea.h"
+
+#include "env-inl.h"
+#include "node_internals.h"
+#include "v8-local-handle.h"
+#include "v8-primitive.h"
+#include "v8-value.h"
+
+#define POSTJECT_SENTINEL_FUSE "NODE_JS_FUSE_fce680ab2cc467b6e072b8b5df1996b2"
+#include "postject-api.h"
+
+#include <tuple>
+
+#if !defined(DISABLE_SINGLE_EXECUTABLE_APPLICATION)
+
+using v8::Local;
+using v8::MaybeLocal;
+using v8::String;
+using v8::Value;
+
+namespace {
+
+bool single_executable_application_code_loaded = false;
+size_t single_executable_application_size = 0;
+const char* single_executable_application_code = nullptr;
```

Comment:
Should these be guarded by a mutex? This is probably fine in a SEA, but in a non-SEA you’ll want to avoid concurrent writes to e.g. `single_executable_application_code_loaded`.

Alternatively, you could just do
```c++
  const char* FindSingleExecutableCode(size_t* size) {
    static std::string_view sea_code = []() {
      // ...
    };
    if (size != nullptr) *size = sea_code.size();
    return sea_code.empty() ? nullptr : sea_code.data();
  }
```

And avoid storing them as top-level global variables entirely.

Reply:
done, @addaleax PTAL!